bot/engine/src/main/kotlin/engine/config/BotDefinitionWrapper.kt

```diff
@@ -37,11 +38,20 @@ internal class BotDefinitionWrapper(val botDefinition: BotDefinition) : BotDefin
     @Volatile
     private var allStories: List<StoryDefinition> = botDefinition.stories
 
-    fun updateStories(configuredStories: List<ConfiguredStoryDefinition>) {
-        logger.debug { "refresh configured stories for ${botDefinition.botId}" }
+    private val dao: StoryDefinitionConfigurationDAO = injector.provide()
+
+    @Volatile
+    private var storyConfRedirections: Map<String, String?> = emptyMap()
```

Comment:
storyConfRedirections should be renamed storyRedirectionsMap with a comment to explain the key and value types